ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -566,8 +565,13 @@ def get_osd_size():
         .get("spec")
         .get("resources")
         .get("requests")
-        .get("storage")[:-2]
+        .get("storage")
     )
+    if not size.isdigit():
```

Comment:
@vavuthu wasn't that fixed here:  https://github.com/red-hat-storage/ocs-ci/pull/4023

As you are talking about LSO deployment it will already go to first `if config.DEPLOYMENT.get("local_storage"):`

But when I am looking to more details to usage of get_deviceset_pvcs then we can improve and consolidate it to something like:

When this is not the case only of UI LSO deployment we should consolidate this code. 

We need to return int, so better to always convert string to int in every return:

```
    sc = get_storage_cluster()
    size = (
        sc.get()
        .get("items")[0]
        .get("spec")
	@@ -566,8 +565,13 @@ def get_osd_size():
        .get("spec")
        .get("resources")
        .get("requests")
        .get("storage")
    )
    if size.isdigit() or config.DEPLOYMENT.get("local_storage"):
        # In the case of UI deployment of LSO cluster, the value in StorageCluster CR
        # is set to 1, so we can not take OSD size from there. For LSO we will return
        # the size from PVC.
        pvc = get_deviceset_pvcs()[0]
        return int(pvc.get()["status"]["capacity"]["storage"][:-2])
    
    else:
        return int(size[:-2])
```

Reply:
Done